### PR TITLE
Why stop gets backpropagated to the mean from the

### PR DESCRIPTION
**Pull request by DaoZhang0123**
I have noticed there is a request[Add a note that stop_gradient in moments does not change the gradient](https://github.com/tensorflow/tensorflow/pull/19609), but I wonder why stop gets backpropagated to the mean from the variance calculation.
In paper[Batch normalization layer (Ioffe and Szegedy, 2014)](https://arxiv.org/abs/1502.03167) Section 3, I have derived the formula:
=(\\sum_{i=1}^m\\frac{\\partial\L}{\\partial\\hat\{x_i}}\\cdot\\frac{-1}{\\sqrt{\\sigma_B^2+\\epsilon}})+\\frac{\\partial\L}{\\partial\\sigma_B^2}\\cdot\\frac{\\sum_{i=1}^m-2(x_i-\\mu_B)}{m})
Gradients of mean are from two parts, one is gradients of variance, so I have some doubts about this.
Besides, I have mentioned }{m}) may be zero, or little, but I can't prove this, is this the answer for my doubt?

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @annarev: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @annarev: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @annarev: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @annarev: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.
